### PR TITLE
github/labeler: Add some more to cephadm, add orchestrator

### PR DESCRIPTION
**Pull request by sebastian-philipp**
Signed-off-by: Sebastian Wagner <sebastian.wagner@suse.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by batrick** [comment]
retest this please

**Review by batrick — APPROVED** [review]

File: .github/labeler.yml [hunk 1]
```diff
@@ -23,11 +23,27 @@ common:
   - src/common
   - src/global
   - src/log
+  - src/python-common/**
```
Comment (tchaikov) [inline .github/labeler.yml:26]: i'd add this to pybind, cephadm, and other consumers of python-common.

Comment (sebastian-philipp) [reply .github/labeler.yml:26]: done

**Review by tchaikov — APPROVED** [review]

**Comment by sebastian-philipp** [comment]
jenkins test make check

**Comment by sebastian-philipp** [comment]
jenkins test make check

**Pull request merged**